modules/ROOT/pages/clustering/setup/analytics-cluster.adoc

```diff
@@ -0,0 +1,272 @@
+:description: This section describes how to deploy a special case Neo4j cluster for analytic queries.
+[role=enterprise-edition]
+[[clustering-analytics]]
+= Deploy an analytics cluster
+
+In analytic use cases, the analytic queries can be intensive enough to warrant running them on separate servers, away from the servers handling the transactional workload.
+This example shows how to set up a cluster to achieve that.
+Information on using Neo4j's graph data science library in a cluster can be found here: link:https://neo4j.com/docs/graph-data-science/current/production-deployment/neo4j-cluster/[Neo4j Graph Data Science Library Manual -> GDS with Neo4j cluster].
+
+The analytics cluster can be set up in two ways, with fault tolerance or without fault tolerance.
```

Comment:
Maybe add a note here that the GDS library does not support fault tolerance, be crystal clear and don't leave anything open to interpretation 🤔

Reply:
Alternatively we can update the documentation at our end to point this out 🤔